### DISCUSSION TITLE
how do I  convert raw to dpmd?

### DISCUSSION THREAD

**Discussion by Wangmumu0403** [deepmodeling/dpdata#253] [Q&A]
```
#!/bin/python

import dpdata
from dpdata import System, LabeledSystem,MultiSystems
d_raw=dpdata.MultiSystems.from_file(file_name='raw',fmt='deepmd/raw')
print(d_raw)
d_raw.to_deepmd_raw('raw')
`d_raw.to_deepmd_npy('dpmd',set_size=372)
```
I have box.raw  coord.raw  energy.raw  force.raw  type_map.raw  type.raw  virial.raw

I use cp2k and I have prepare the raw file, now I need to convert it to dpmd/, 
I have trouble in writing this datatransform.py bash.
could you help me ?

**Comment by Wangmumu0403** [comment]
I have found the solution. 
the developer have provided the bash --- raw_to_set.sh
thankyou!

**Reply by RobinMAJ** [reply]
Can you share the "raw_to_set.sh" file? thank you very much
